PennMobile/src/main/java/com/pennapps/labs/pennmobile/api/fragments/LoginWebviewFragment.kt

```diff
@@ -260,6 +267,20 @@ class LoginWebviewFragment : Fragment() {
         }
     }
 
+    private fun sendNotifToken() {
+        val mNotificationAPI = MainActivity.notificationAPIInstance
+
+        val bearerToken = "Bearer " + sp.getString(getString(R.string.access_token), "").toString()
+        val notifToken = sp.getString(getString(R.string.notification_token), "").toString()
+
+        Log.d("Notification Token", notifToken)
+        val notGuest = !sp.getBoolean(mActivity.getString(R.string.guest_mode), false)
```

Comment:
Confused about the logic here. If you are not a guest, then shouldn't you remove the ! mark or change the false to true? I feel you you are negating the negation

Reply:
Wait, you can just remove ! and change false to true.